source/tutorials/nix-language.md

```diff
@@ -292,7 +304,7 @@ Nix language data types *without functions* work just like their counterparts in
   <th>JSON</th>
 </tr>
 <tr>
-<td>
+<td style="padding-right: 1em">
```

Comment:
Won't this make the table asymmetric? I think that change is unrelated.

Reply:
It doesn't, the web browser will realign the rest.

![image](https://user-images.githubusercontent.com/18437312/206521276-c2475d7c-3932-4c10-85e9-034a5c63639e.png)

But yes, I forgot to make a separate commit for this change.